docs/table_design/Data_distribution.md

```diff
@@ -529,28 +529,54 @@ Buckets reflect how data files are actually organized in StarRocks.
 
 - How to set the number of buckets at table creation
 
-  - Method 1: automatically set the number of buckets
+  - Method 1: automatically set the number of buckets (recommended)
 
-    Since v2.5.7, StarRocks can automatically set the number of buckets based on machine resources and data volume for a partition.
+    - Table configured with hash bucketing
 
-    Example:
+      Since v2.5.7, StarRocks can automatically set the number of buckets based on machine resources and data volume for a partition at table creation.
+      > **NOTICE**
+      >
+      > If the raw data size of a partition exceeds 100 GB, we recommend that you manually configure the number of buckets using the Method 2.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/75470710-693d-4967-9bdf-30ec20f2441c)
Remove "the"